services/port_manager/src/main/java/com/futurewei/alcor/portmanager/controller/PortController.java

```diff
@@ -221,4 +221,16 @@ public RouterUpdateInfo updateL3Neighbors(@PathVariable("project_id") String pro
         checkRouterSubnetUpdateInfo(routerUpdateInfo);
         return portService.updateL3Neighbors(projectId, routerUpdateInfo);
     }
+
+    @Rbac(resource ="port")
+    @GetMapping({"/project/{project_id}/subnet-port-number/{subnet_id}", "v4/{project_id}/subnet-port-number/{subnet_id}"})
+    @DurationStatistics
+    public int getSubnetPortNumber(@PathVariable("project_id") String projectId,
+                                             @PathVariable("subnet_id") String subnetId) throws Exception {
+        if (StringUtil.isNullOrEmpty(subnetId)) {
```

Comment:
@chenpiaoping do you think function name `getSubnetPortCount` would be better? Same as the API name: `subnet-port-count`.

Reply:
agree